static/js/tools/stat_var/dataset_selector.tsx

```diff
@@ -0,0 +1,231 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Component to select a source and dataset.
+ */
+
+import axios from "axios";
+import React, { useEffect, useState } from "react";
+import { Card, Container, CustomInput } from "reactstrap";
+
+import { NamedPlace } from "../../shared/types";
+
+const PREFIX = "dataset-selector";
```

Comment:
nit: maybe CSS_PREFIX or CSS_SELECTOR_PREFIX or something mentioning what the prefix is for

Reply:
changed to CSS_PREFIX